VENDOR_NEUTRALITY.md

```diff
@@ -0,0 +1,33 @@
+# Metal3-io vendor neutrality declaration
+
+As a Cloud Native Computing Foundation project, Metal3-io will keep its
+[vendor neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/)
+
+**Hardware vendor neutrality:**
+As it is stated in the [documentation](https://book.metal3.io/) Metal3-io is a
+CNCF project with the goal to provide bare metal life cycle management
+functionality regardless of the vendor of the hardware. The only requirements
+are that the machines are equipped with a [BMC](https://www.servethehome.com/explaining-the-baseboard-management-controller-or-bmc-in-servers/)
+and support either [Redfish](https://redfish.dmtf.org/) or [IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface)
+protocols. Metal3 also supports vendor specific versions of the aforementioned
+protocols but that support is on top of the standard protocols and the reason
```

Comment:
Strictly speaking, neither iLO4 nor iRMC are versions of IPMI or Redfish. Maybe we should say something along the lines of focusing on industry-wide protocols, while only allowing vendor drivers for hardware that does not support them and only within the feature scope of the common protocols.

Reply:
I have tried to integrate your comment into the text, I hope it is okay like this.